Source/items.cpp

```diff
@@ -4256,18 +4305,38 @@ void CreateMagicWeapon(int x, int y, int imisc, int icurs, BOOL sendmsg, BOOL de
 	BOOL done;
 
 	done = FALSE;
+#ifdef HELLFIRE
+	int imid;
+	if (imisc == 10)
+		imid = 23;

```

Comment:
```suggestion
		imid = IMISC_STAFF;
```

Reply:
my bad, forgot to apply the enums because this function was such a nightmare haha

Reply:
You should have looked at my min diff version :)